core-bundle/src/Resources/contao/classes/Backend.php

```diff
@@ -263,10 +262,10 @@ public static function handleRunOnce()
 
 			if (!unlink($file))
 			{
-				throw new \Exception("The file $strRelpath cannot be deleted. Please remove the file manually and correct the file permission settings on your server.");
+				throw new \Exception('The file ' . $strRelpath . ' cannot be deleted. Please remove the file manually and correct the file permission settings on your server.');
```

Comment:
this looks like an unrelated change

Reply:
I changed the quotes for the log message in the line below and couldn't resist the urge to "correct" the quotes in this line as well. It looked weird having pretty much the same construct within 2 lines but with different syntax.

Reply:
Since PHP 8, using double quotes and interpolation is more performant than single quotes and concatenation, so I guess the change should be reverted. I did not find the link though, maybe @m-vo can help?

Reply:
For me, it is more about consistency than performance, as the other log actions (even in the same file) use string concatenation or `sprintf`. 

If interpolation is more performant, we should probably check the whole system for such optimisations in a follow up pull request.

I can of course revert the change if that is preferred.

Reply:
Interpolation is the fastest (https://3v4l.org/3MZYb#v8.1.1) but it doesn't matter much, i.e. it's a micro optimization that probably doesn't matter anywhere.

However, string interpolation beats sprintf/concatenation in terms of readability. That's a personal opinion of course, but it's backed by literally all modern languages introducing it. :upside_down_face:

Reply:
(Dang it, three times edit for 4 lines of message - now it's right hopefully. :sweat_smile:)

Reply:
Thanks for the interesting check. I'd still vote for a consistent style for now – if there should be changes, a separate issue to discuss and agree on a style for string concatenation is probably the proper way to go.

Reply:
We could not find the link to the document anymore. It might have been deleted or even never existed 😄, so we will stick to Symfony's existing "prefer string concatenation over interpolation" rule.